sunxspex/brem_49.py

```diff
@@ -0,0 +1,43 @@
+import numpy as np 
+from acgaunt import Acgaunt
+from astropy import units as u
+
+@u.quantity_input(energy=u.keV, temperature=u.keV)
+def brem_49(energy, temperature):
+	"""
+	The function calculates the optically thin continuum thermal bremmstrahlung
+	photon flux incident on the Earth from an isothermal plasma on the Sun.
+	Normalization is for an emission measure on the sun of 1.e49cm^-3
+
+	Catagory Spectra
+	
+	Parameters
+	----------
+	energy : `~astropy.Quantity`
+		energy array in keV
+	temperature : `~astropy.Quantity` 
+		 scaler plasma temperature in keV
+
+	Returns
+	-------
+	Differential photon flux in units of photons/(cm2 s keV) per (1e49 cm-3 emission measure)
+
+	Examples
+	--------
+	>>> flux = brem_49(energy, kt)
+
+	Notes
+	-----
+	This is currently implemented following the IDL version
+
+	"""
+	if not isinstance(temperature.value, (int, float)):
+		raise ValueError('Only one temperature value is allowed - must be a float or int')
+
+	acgaunt = Acgaunt(energy.to(u.angstrom, equivalencies=u.spectral()), temperature.to(u.Kelvin, equivalencies=u.temperature_energy()))
```

Comment:
I think the equivalencies should be moved to the decorator of ```Acgaunt```, rather than being required in the call.  What do you think?